docs/signing-spec.md

```diff
@@ -46,7 +91,10 @@ Examples of proxy signers:
 - CI systems
 - Users who transfer bindles from one Bindle server to another
 - Import/export tools
+- Proxy servers
 
+While this is implicit in the definitions, we should make it _explicit_ that a Bindle server can use the same key to sign as a host (for bindles it hosts) and as a proxy (for pass-through and cache features).
```

Comment:
Up above it says that an invoice cannot be signed by the same key twice. How does that work here?

Reply:
Ah, yeah... I will clarify that I mean it can sign one invoice as a host while another invoice it could sign as a proxy... using the same key for both roles.